modelon/impact/client/jupyterhub/authorize.py

```diff
@@ -17,9 +17,13 @@ def _get_jupyter_token(credential_manager, interactive, context=None):
 
 
 def authorize(uri, interactive, context=None, credential_manager=None, service=None):
+    # Checking for deprecated environment variable
+    # MODELON_IMPACT_JUPYTERHUB_CLIENT_API_TOKEN for backward compatibility
     credential_manager = credential_manager or CredentialManager(
         file_id="jupyterhub-api.key",
-        env_name="MODELON_IMPACT_JUPYTERHUB_CLIENT_API_TOKEN",
+        env_name="JUPYTERHUB_API_TOKEN"
+        if os.environ.get("JUPYTERHUB_API_TOKEN")
+        else "MODELON_IMPACT_JUPYTERHUB_CLIENT_API_TOKEN",
```

Comment:
I suggest to deprecate the use of MODELON_IMPACT_JUPYTERHUB_CLIENT_API_TOKEN. We'll be releasing a major version update anyhow and I see no reason for having two different env vars for the same thing.